pipeline/data_merging/brca_pseudonym_generator.py

```diff
@@ -77,7 +77,7 @@ def from_field_or_compute(row):
         if not computed:
             cDNA = cdna_from_cdna_field(row)
             if cDNA is None:
-                logging.info("No calculable cDNA for row: " + str(row))
+                logging.info("No calculable cDNA for row: {row}")
```

Comment:
need the f in front in order for the string interpolation to work: `logging.info(f"No calculable cDNA for row: {row}")`